.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-      - uses: msys2/setup-msys2@v2
+      - uses: msys2/setup-msys2@v2.22.0
```

Comment:
Hi, @sloede,
I try to downgrade setup-msys2 here to get older version of msys2, because it is not possible to choose gcc version in msys2, only by installing older version of msys2 itself. I'm not really good in CI stuff, I used `msys2/setup-msys2@v2.22.0`, but it still uses newest version of gcc. If you have experience with downgrading GitHub Action version, am I doing it right? If you haven't, do not worry, I will find out this

Reply:
using `msys2/setup-msys2@v2.22.0` will only change the version of the *GitHub action* script, i.e., the version of https://github.com/msys2/setup-msys2. What you really want is a specific version of `mingw-w64-x86_64-toolchain` in line 70 below

Reply:
Maybe this is not really possible:
https://stackoverflow.com/a/33986757
https://github.com/msys2/msys2.github.io/issues/205

Reply:
older msys2/setup-msys2@v2.22.0 installs older msys2 with older gcc, I recognized, that there is an `update: true` below, now I was able to install older version of msys with gcc 13.2, should work

Reply:
However, if this only affects Windows testing, I'd be ok with accepting a failing windows test, if it only concerns the self-built version of openfhe and not the Yggdrasil-provided ones

Reply:
Am I right, that we are not choosing gcc version for Yggdrasil, is it done automatically?

Reply:
> older msys2/setup-msys2@v2.22.0 installs older msys2 with older gcc, I recognized, that there is an `update: true` below, now I was able to install older version of msys with gcc 13.2, should work

Wow, this is great! In that case, maybe just do not update for now, but add a TODO comment that we should fix this once OpenFHE 1.2.1 is available and hopefully fixes this issue.

Reply:
> Am I right, that we are not choosing gcc version for Yggdrasil, is it done automatically?

Yes. I think we specify a rather old version usually to ensure compatibility the greatest possible variety of GCC versions.

Reply:
> > older msys2/setup-msys2@v2.22.0 installs older msys2 with older gcc, I recognized, that there is an `update: true` below, now I was able to install older version of msys with gcc 13.2, should work
> 
> Wow, this is great! In that case, maybe just do not update for now, but add a TODO comment that we should fix this once OpenFHE 1.2.1 is available and hopefully fixes this issue.

Okay! I'll do!